syntax = "proto3";

package Persistance;

import "Vector.proto";

message ScatterSample {
    int32 scene_setup_id = 1;
	Persistance.Vector3 point = 2;
	Persistance.Vector3 view_direction = 3;
}